packages/manager/core/generator/app/templates/src/components/layout-helpers/Dashboard/Dashboard.tsx

```diff
@@ -22,23 +20,25 @@ export type DashboardLayoutProps = {
 };
 
 const Dashboard: React.FC<DashboardLayoutProps> = ({ tabs }) => {
-  const [panel, setActivePanel] = useState('');
+  const [activeTab, setActiveTab] = useState<string>('');
   const location = useLocation();
   const navigate = useNavigate();
 
   useEffect(() => {
-    const activeTab = tabs.find((tab) => tab.to === location.pathname);
-    if (activeTab) {
-      setActivePanel(activeTab.name);
+    const activeTabFromLocation = tabs.find(
+      (tab) => tab.to === location.pathname,
+    );
+    if (activeTabFromLocation) {
+      setActiveTab(activeTabFromLocation.to);
     } else {
-      setActivePanel(tabs[0].name);
+      setActiveTab('');
       navigate(`${tabs[0].to}`);
     }
-  }, [location.pathname]);
+  }, [location.pathname, tabs, navigate]);
```

Comment:
if you put tabs and navigate, it's like you didn't use a `useEffect` here

Reply:
sure can delete it please ?